### ISSUE TITLE
Rename broadcast, and implement a lagging broadcast channel

### ISSUE THREAD

**Issue by austinjones** [austinjones/postage-rs#21]
I'd like to support the lagging broadcast channel that tokio provides, so postage can be fully compatible with existing tokio code.

I think the current channel is useful, but it needs a new name.

**Comment by zeenix** [comment]
> I'd like to support the lagging broadcast channel that tokio provides, so postage can be fully compatible with existing tokio code.

Tokio's behaviour to me (and some other Rust devs I've talked to) seems a bit wrong, to just drop a message if a receiver is lagging. However, if it's an optional feature, people can just choose the behavior they expect.

**Comment by austinjones** [comment]
Yeah, I think the tokio behavior is there because writing a broadcast channel with back-pressure is **difficult**.

The only contetxt that dropping might make sense is in audio/video processing apps (for frames or samples).  But in general apps it can cause some wild bugs that are really hard to reproduce.
